json-schema/src/main/java/com/regnosys/rosetta/generator/jsonschema/JsonSchemaTranslator.xtend

```diff
@@ -0,0 +1,32 @@
+package com.regnosys.rosetta.generator.jsonschema
+
+import com.regnosys.rosetta.generator.object.ExpandedType
+
+class JsonSchemaTranslator {
+				
+	static def toJsonSchemaBasicType(String typename) {
+		switch typename {
+			case 'string',
+			case 'time',
+			case 'date',
+			case 'dateTime',
+			case 'zonedDateTime',
```

Comment:
Dates and times could be improved. See https://json-schema.org/understanding-json-schema/reference/string#dates-and-times.
(do note the draft version though)